roles/kubernetes/loadbalancer/metallb/tasks/main.yaml

```diff
@@ -10,6 +10,9 @@
 #       Reference URL : https://docs.ansible.com/ansible/latest/inventory/implicit_localhost.html
 #
 - name: Setting up MetalLB loadbalancer
+  tags:
+    - install
+    - update
```

Comment:
Tags and conditionals after blocks

Reply:
The order was updated to follow ansible conventions based off this rule: 
https://ansible.readthedocs.io/projects/lint/rules/key-order/